lib/tictactoe.rb

```diff
@@ -39,6 +50,29 @@ def show_board(board_spaces)
   end
 
 
+  def check_for_winner?(board)
+    winner_symbol = ""
+    for win_combo in WIN_COMBOS do 
+      win_index_1 = win_combo[0]
+      win_index_2 = win_combo[1]
+      win_index_3 = win_combo[2]
+
+      winning_position_1 = board[win_index_1]
+      winning_position_2 = board[win_index_2] 
+      winning_position_3 = board[win_index_3] 
```

Comment:
instead of having multiple win_index and win_positions variables. Can you try putting them into another list.

```
for win_combo in WIN_COMBOS do
   row = Array.new
   for index in win_combo do
     row.append( board[ win_combo[ index ] ]
     ...
        
  end
end
```

Reply:
👍🏽